app/views/layout.njk

```diff
@@ -12,4 +12,4 @@
     <script src="https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.6/dist/semantic.min.js"></script>
     {% block js %}{% endblock %}
   </body>
-</html>
+</html>
```

Comment:
Can you get rid of this change? Generally you want to leave a blank line at the end of every file.